### PR TITLE
fix(mount): Fix low level fuse options in help

### PR DESCRIPTION
**Pull request by rolysr**
Previously, the help message for the client was not showing some options that were available in the man page for sfsmount. This commit fixes that issue.

### CODE REVIEW

**Review by lgsilva3087 — DISMISSED** [review]
LGTM

**Review by dmga44 — APPROVED** [review]
LGTM

**Review by aNeutrino — CHANGES_REQUESTED** [review]

File: src/mount/fuse/mount_config.cc [hunk 1]
```diff
@@ -303,6 +303,10 @@ void usage(const char *progname) {
 	printf("\nFUSE options:\n");
 	fuse_lowlevel_help();
```
[the remark this answers is not in the document]
Comment (rolysr) [reply src/mount/fuse/mount_config.cc:304]: Done

**Review by dmga44 — DISMISSED** [review]

File: src/mount/fuse/mount_config.cc [hunk 2]
```diff
@@ -303,30 +303,48 @@ void usage(const char *progname) {
...
+				"bits to take effect\n"
+"    -o dev                 interpret character or block special devices on the "
+				"filesystem\n"
+"    -o nodev               do not interpret character or block special devices on "
+				"the file system\n"
+"    -o exec                permit execution of binaries\n"
+"    -o noexec              do  not  permit direct execution of any binaries on the " 
+				"mounted filesystem\n"
+"    -o max_write=N         set the maximum number of bytes  in  a  single  write  "
+				"operation. The  default  is 128kbytes. Note, that due to various "
```
Comment (ralcolea) [inline src/mount/fuse/mount_config.cc:320]: Use `KB` or `kB` to refer to kilobytes. Please, extend to the rest of the documentation.

Usage example:
`The default is 128KB.`

**Review by ralcolea — APPROVED** [review]
LGTM. Please, see some minor suggestions I shared for your consideration.

File: src/mount/fuse/mount_config.cc [hunk 3]
```diff
@@ -303,30 +303,48 @@ void usage(const char *progname) {
...
+"    -o rw                  mount the filesystem read-write\n"
+"    -o ro                  mount the filesystem read-only.\n"
+"    -o suid                allow set-user-identifier or set-group-identifier "
+				"bits to take effect\n"
+"    -o nosuid              do not allow set-user-identifier or set-group-identifier " 
+				"bits to take effect\n"
+"    -o dev                 interpret character or block special devices on the "
+				"filesystem\n"
+"    -o nodev               do not interpret character or block special devices on "
+				"the file system\n"
```
[the remark this answers is not in the document]
Comment (rolysr) [reply src/mount/fuse/mount_config.cc]: Done

File: src/mount/fuse/mount_config.cc [hunk 4]
```diff
@@ -303,30 +303,48 @@ void usage(const char *progname) {
...
-	"    -o async_dio \n"
-	"    -o no_async_dio \n"
-	"    -o writeback_cache \n"
-	"    -o no_writeback_cache \n"
-	"    -o time_gran=N\n"
+"    -o rw                  mount the filesystem read-write\n"
+"    -o ro                  mount the filesystem read-only.\n"
+"    -o suid                allow set-user-identifier or set-group-identifier "
+				"bits to take effect\n"
+"    -o nosuid              do not allow set-user-identifier or set-group-identifier "
```
[the remark this answers is not in the document]
Comment (rolysr) [reply src/mount/fuse/mount_config.cc]: Done

[the remark this answers is not in the document]
Comment (rolysr) [reply src/mount/fuse/mount_config.cc]: Done

File: src/mount/fuse/mount_config.cc [hunk 5]
```diff
@@ -303,30 +303,48 @@ void usage(const char *progname) {
...
+"    -o nosuid              do not allow set-user-identifier or set-group-identifier " 
+				"bits to take effect\n"
+"    -o dev                 interpret character or block special devices on the "
+				"filesystem\n"
+"    -o nodev               do not interpret character or block special devices on "
+				"the file system\n"
+"    -o exec                permit execution of binaries\n"
+"    -o noexec              do  not  permit direct execution of any binaries on the " 
+				"mounted filesystem\n"
+"    -o max_write=N         set the maximum number of bytes  in  a  single  write  "
```
[the remark this answers is not in the document]
Comment (rolysr) [reply src/mount/fuse/mount_config.cc]: Fixed

File: src/mount/fuse/mount_config.cc (hunk 2 above)
Comment (rolysr) [reply src/mount/fuse/mount_config.cc]: Done

**Review by ralcolea — APPROVED** [review]
LGTM

**Review by lgsilva3087 — APPROVED** [review]
LGTM

**Review by aNeutrino — APPROVED** [review]

**Pull request merged**